extension/android_test/src/test/java/org/pytorch/executorch/EValueTest.java

```diff
@@ -9,29 +9,25 @@
 package org.pytorch.executorch;
 
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
 
-import com.facebook.jni.annotations.DoNotStrip;
-
-import java.util.List;
-import java.util.ArrayList;
 import java.util.Arrays;
-import java.util.Locale;
 import java.util.Optional;
-
-import org.pytorch.executorch.Tensor.Tensor_int64;
-import org.pytorch.executorch.annotations.Experimental;
-
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
 /** Unit tests for {@link EValue}. */
 @RunWith(JUnit4.class)
 public class EValueTest {
+  private static final int TYPE_CODE_NONE = 0;
+  private static final int TYPE_CODE_TENSOR = 1;
+  private static final int TYPE_CODE_STRING = 2;
+  private static final int TYPE_CODE_DOUBLE = 3;
+  private static final int TYPE_CODE_INT = 4;
+  private static final int TYPE_CODE_BOOL = 5;
```

Comment:
maybe we can just expose the constants in EValue as public instead of copying this here